jbpm-human-task/jbpm-human-task-core/src/test/java/org/jbpm/services/task/jaxb/JaxbTaskSerializationTest.java

```diff
@@ -282,7 +282,11 @@ public void setTaskPropertyCommandTest() throws Exception {
        for( Object [] data : testData ) {
            int property = (Integer) data[0];
            cmd = new SetTaskPropertyCommand(taskId, userId, property, data[1]);
-           cmd.execute(mockContext);
+           try {
+               cmd.execute(mockContext);
+           } catch (IllegalArgumentException e) {
```

Comment:
If the aim is just to avoid possible exception from `doCallbackUserOperation` method, the exception should be asserted whether it's really the exact one we are counting with; i.e. this piece of code can swallow a different `IllegalArgumentException`.

Reply:
done, now it asserts on the expected message